src/vscode-bicep/src/test/utils/assert.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import { Range } from "vscode";
+
+export function expectDefined<T>(value: T | undefined): asserts value is T {
+  expect(value).toBeDefined();
+}
+
+export function expectRange(
+  range: Range,
+  startLine: number,
+  startCharacter: number,
+  endLine: number,
+  endCharacter: number
+): void {
+  expect(range.start.line).toBe(startLine);
+  expect(range.start.character).toBe(startCharacter);
+  expect(range.end.line).toBe(endLine);
+  expect(range.end.character).toBe(endCharacter);
```

Comment:
Will jest group these assertions or just fail on the first one? e.g. if startLine and startCharacter both change, will the error message show both changes or just the first?

Reply:
Good catch. Changed it to use `toMatchObject` which does exactly what we want.